service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/HearingBookingDetailsControllerTest.java

```diff
@@ -93,17 +92,19 @@ private HearingBooking createHearing(LocalDate hearingDate) {
     }
 
     private AboutToStartOrSubmitCallbackResponse makeRequest(HearingBooking hearingDetail) throws Exception {
-        HashMap<String, Object> map = objectMapper.readValue(objectMapper.writeValueAsString(hearingDetail),
-            new TypeReference<Map<String, Object>>() {
+        Map<String, Object> map = objectMapper.readValue(objectMapper.writeValueAsString(hearingDetail),
```

Comment:
Good change although I suggest another PR. It has nothing to do with Jackson upgrade.

Reply:
This was fixing issues that then occurred due to the upgrade